src/items.rs

```diff
@@ -1905,12 +1905,22 @@ fn get_missing_arg_comments(
 
 impl Rewrite for ast::Param {
     fn rewrite(&self, context: &RewriteContext<'_>, shape: Shape) -> Option<String> {
+        let mut param_attrs_result = self
+            .attrs
+            .to_vec()
+            .rewrite(context, Shape::legacy(shape.width, shape.indent))?;
+        if !self.attrs.is_empty() {
+            param_attrs_result = format!("{} ", param_attrs_result);
```

Comment:
Unfortunately, this does not always work. If the attribute or the parameter is too long, then when combined the line may exceed the `max_width`. We should use `combine_strs_with_missing_comments` here and in other places instead.

Reply:
Makes sense, will update accordingly 👍

Reply:
Have made the updates, though I do have an inline question below regarding the arg value for `allow_extend`